src/apps/project-home/AssignmentsView/AssignmentDetail/AssignmentDetail.tsx

```diff
@@ -1,5 +1,5 @@
 import type { Context, Translations, UserProfile, Document } from 'src/Types';
-import { Pencil, Trash } from '@phosphor-icons/react';
+import { Pencil, Trash } from '@phosphor-icons/react/dist/ssr';
```

Comment:
I don't know why the autocomplete sometimes do this. But the correct import is `@phosphor-icons/react`. Not sure if it doesn't matter in practice, or if this would indeed possibly break when used in client-only mode rather than SSR...